GetIntrinsic.js

```diff
@@ -7,7 +7,24 @@
 
 var undefined;
 
+var $Function = Function;
 var $TypeError = TypeError;
+var $SyntaxError = SyntaxError;
+
+var getESNextFunction = function (name, syntax) {
+	try {
+		return $Function("'use strict'; return " + syntax + ';')();
```

Comment:
It is not really a practical option to use `Function` in this file; that ensures that nobody can ever use this library in a CSP environment (ie, it's a breaking change).

Reply:
That’s why there’s the `require('./GetIntrinsic' + name + '.js')` fallback.

Reply:
Right, but the mere *presence* of `Function` will cause the entire app not to load in a CSP env, as far as I understand.

Reply:
That’s not how **CSP** works.

**CSP** causes `Function(...)` to throw an `EvalError`.

Reply:
hmm, ok.